integration-tests/main/src/test/java/io/quarkus/it/main/RestClientTestCase.java

```diff
@@ -18,6 +19,10 @@
 
     public static final String HEADER_NAME = "header-name";
 
+    // this is required, otherwise the test.url system property is never set
+    @TestHTTPResource
```

Comment:
I think it would be better to change the code to always set the property.

Reply:
Fine by me, but then where/how?